internal/engine/upper.go

```diff
@@ -935,6 +935,7 @@ func handleDockerComposeLogAction(state *store.EngineState, action DockerCompose
 
 	dcState, _ := ms.ResourceState.(dockercompose.State)
 	ms.ResourceState = dcState.WithCurrentLog(model.AppendLog(dcState.CurrentLog, action, state.LogTimestamps))
+	state.Log = model.AppendLog(dcState.CurrentLog, action, state.LogTimestamps)
```

Comment:
IIUC this line is currently replacing the full tilt log with the runtime log of one DC service, and this should be `ms.CombinedLog`, not `state.Log`.

Reply:
Ohh yes, thank you